docs/en/sql-reference/sql-functions/string-functions/bit_length.md

```diff
@@ -0,0 +1,54 @@
+---
+{
+    "title": "bit_length",
+    "language": "en"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# bit_length
+## Description
+### Syntax
+
+'INT bit_length (VARCHAR str)'
+
+
+Return length of argument in bits。
+
+## example
+
+```
+mysql> select bit_length("abc");
++---------------+
+| length('abc') |
```

Comment:
```suggestion
| bit_length('abc') |
```

Same as others

Reply:
And you need to add this doc to the `docs/.vuepress/sidebar/*.js`

Reply:
Done